### DISCUSSION TITLE
Strava token

### DISCUSSION THREAD

**Discussion by QQ7ita** [mytourbook/mytourbook#1341] [Q&A]
Hi,
from MTB I can no longer upload activities to Strava, it tells me the token expired on the 10th of this month.
If I try in Tools - Preferences - Cloud - Strava the command “Connect with Strava” I get a Strava window where it asks me to confirm the connection but then it always returns error, I've been trying for a couple of days but the situation doesn't change. I also tried with another browser from pc and also from smartphone.
I also tried deleting MTB on Strava from My Apps but I still can't connect.

**Comment by FJBDev** [comment]
This is a known issue that will be fixed in 24.5

See more details [here](https://github.com/mytourbook/mytourbook/issues/1337)
